task/mongo-optimization-task.js

```diff
@@ -15,7 +15,14 @@ const ObjectId = require('mongodb').ObjectID;
  * Test timeout is increased to 60sec for the function.
  * */
 async function before(db) {
-    await db.collection('opportunities').createIndex({'initiativeId': 1});
+    await db.collection('opportunities').createIndex({
+        'initiativeId': 1,
+        'contacts.questions.category_id': 1,
+    });
+    await db.collection('clientCriteria').createIndex({
```

Comment:
good idea but order of fields in compound index is important too. Please try both